test/test_requirements.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import csv
+import os
+import re
+import subprocess
+from importlib.metadata import version
+
+REQ_TXT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "requirements.txt")
+)
+REQ_CSV = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "requirements.csv")
+)
+DOC_TXT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "doc", "requirements.txt")
+)
+DOC_CSV = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "doc", "requirements.csv")
+)
+
+CACHE_PATH = os.path.join(os.path.expanduser("~"), ".cache", "cve-bin-tool")
+
+CACHE_CSV = os.path.join(CACHE_PATH, "requirements.csv")
+
+HTML_DEP_PATH = os.path.abspath(
+    os.path.join(
+        os.path.dirname(__file__),
+        "..",
+        "cve_bin_tool",
+        "output_engine",
+        "html_reports",
+        "js",
+    )
+)
+HTML_DEP_CSV = os.path.join(HTML_DEP_PATH, "dependencies.csv")
+
+# Dependencies that currently have CVEs
+# Remove from the list once they are updated
+ALLOWED_PACKAGES = ["reportlab"]
+
+
+def get_out_of_sync_packages(csvfile, txtfile):
+
+    out_of_sync_packages = []
+    csv_package_names = []
+
+    with open(csvfile, "r") as csvf, open(txtfile, "r") as txt:
```

Comment:
```suggestion
    with open(csvfile) as csvf, open(txtfile) as txt:
```
You can omit "r" mode when opening files and some formatting tools even remove this (see #1147 "Redundant open modes").